lib/app.jsx

```diff
@@ -151,7 +151,12 @@ export const App = connect(
         .on('index', this.onNotesIndex)
         .on('update', this.onNoteUpdate)
         .on('update', debounce(this.onNotesIndex, 200, { maxWait: 1000 })) // refresh notes list
-        .on('remove', this.onNoteRemoved);
+        .on('remove', this.onNoteRemoved)
+        .subscribe(noteId =>
```

Comment:
I'm happy to change this to be something analogous to `.on( 'beforeNetworkApplyChange', ...)`.

The result will probably mean passing a function in that `onNoteBeforeRemoteUpdate` will need to call with the latest data.